CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 * Adds documentation for unquotes `~( )`.
 * Adds documentation for laziness and codata primitives: `Lazy`, `Inf`, `Delay`,
   and `Force`.
+* Adds documentation for `Not`.
```

Comment:
I think, this line is added in the wrong section of the changelog. This section is for changes in REPL, and REPL has special functionality for showing documentation for syntax constructions, and this section is for such additions. You've added documentation to the source code, not to the REPL. I'm even not sure that your change should go into the changelog, since it just adds what should have been there.

Reply:
OK.  I wasn't sure, but saw there were doc changes in the REPL section.  I didn't see any other section for docu,entation.  I can change the PR or withdraw it and make a new one (I think).

Reply:
It's better to change this PR